Update user-env.md (#264)
icub-tech-iit/documentation: docs/icub_operating_systems/icubos/user-env.md

## Online updated version

You can find the updated default .bashrc_iCub file online.
There are two versions of the iCub environment file:

- [`bashrc_iCub_superbuild`](https://raw.githubusercontent.com/icub-tech-iit/icub-os-files/master/user-environment/bashrc_iCub_superbuild) - a special version of icub user environment, in case of compilation of the repository [ROBOTOLOGY SUPERBUILD](https://github.com/robotology/robotology-superbuild)

> 

Reviewer: ```suggestion
```

## Required configuration

> In order to customize the above user environment you should manually make some changes:

Reviewer: ```suggestion
To customize the above user environment, you ought to manually make some changes:
```
